r2r/base/abstractions/exception.py

```diff
@@ -0,0 +1,16 @@
+from typing import Any, Optional
+
+
+class R2RException(Exception):
```

Comment:
Nice. Might circle back to make this support unpacking HTTP/API errors fully

Reply:
cheers